aspnetcore/includes/net-core-prereqs-windows.md

```diff
@@ -1,3 +1,6 @@
-[Visual Studio for Windows](https://www.microsoft.com/net/download/windows)
-* **ASP.NET and web development** workload
-* **.NET Core cross-platform development** workload
+[Visual Studio for Windows](https://www.microsoft.com/net/download/windows).
+Select the **ASP.NET and web development** workload.
```

Comment:
If you want the bare return for this line, add the two spaces at the end of Line 1.